apps/send-func/src/domain/telemetry.ts

```diff
@@ -0,0 +1,11 @@
+export interface TelemetryService {
+  trackEvent(name: string, properties?: object): void;
+}
+
+export enum TelemetryEventName {
+  SEND_AAR_ATTACHMENT_SERVER_ERROR = "io.com.send.aar.send_aar_attachment_server_error",
```

Comment:
I think we can remove the duplication here, there is no need to repeat `send` and `aar`
```suggestion
  SEND_AAR_ATTACHMENT_SERVER_ERROR = "io.com.send.aar.attachment_server_error",
```